src/main/java/com/ibm/cloudant/kafka/common/InterfaceConst.java

```diff
@@ -1,18 +1,16 @@
-/*******************************************************************************
-* Copyright (c) 2016 IBM Corp.
-*
-* Licensed under the Apache License, Version 2.0 (the "License");
-* you may not use this file except in compliance with the License.
-* You may obtain a copy of the License at
-*
-*    http://www.apache.org/licenses/LICENSE-2.0
-*
-* Unless required by applicable law or agreed to in writing, software
-* distributed under the License is distributed on an "AS IS" BASIS,
-* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-* See the License for the specific language governing permissions and
-* limitations under the License.
-*******************************************************************************/
+/*
+ * Copyright © 2016, 2018 IBM Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
 package com.ibm.cloudant.kafka.common;
 
 public class InterfaceConst {
```

Comment:
Based on the design in `kafka-connect-jdbc` and other kafka connectors (e.g. hfs) , I'm wondering if the constants in this class should be moved to `CloudantSinkConnectorConfig ` since it extends `AbstractConfig`.  

[JdbcSinkConfig.java in kafka-connect-jdbc](https://github.com/confluentinc/kafka-connect-jdbc/blob/master/src/main/java/io/confluent/connect/jdbc/sink/JdbcSinkConfig.java)

Reply:
Well some of them apply only to the source connector so it would seem weird to define them in the `CloudantSinkConnectorConfig`.
The shared ones from here are referenced in both the `CloudantSourceConnectorConfig` and `CloudantSinkConnectorConfig`.

I think this is the correct place based on the current code layout, but I'm definitely not ruling out some sort of tidy up in future. There are loads of unused constants in the `com.ibm.cloudant.kafka.common.CloudantConst` class for example too.